Leanplum/Common/LPConstants.h

```diff
@@ -7,17 +7,36 @@
 //
 
 #import <Foundation/Foundation.h>
+#import "LPApiConstants.h"
 
 
 #define LEANPLUM_SDK_VERSION @"2.4.1"
 #define LEANPLUM_API_VERSION @"1.0.6"
 #define LEANPLUM_CLIENT @"ios"
 #define LEANPLUM_SUPPORTED_ENCODING @"gzip"
+
+// Can upload up to 100 files or 50 MB per request.
+#define MAX_UPLOAD_BATCH_SIZES (50 * (1 << 20))
```

Comment:
We want to be moving these into an `LPSDKConfig` or something similar. Do you want to hold off on that for later?

Reply:
yea, i think we can do find this appropriate location later if we think so. Currently all the constants are here.